.github/workflows/nightlies.yml

```diff
@@ -0,0 +1,310 @@
+# Github Action to build nightly releases
+#
+# This script builds and packages a release for Linux, Windows, OSX and FreeBSD
+# using current master. The generated archives are published as the current
+# nightly build on the Gitbub release page.
+#
+# Job overview:
+# 1. Generates the documentation included in each release
+# 2. Builds the actual release (using a matrix over all targets)
+# 3. Publishes all artifacts from (2) to the release page on GitHub
+
+name: build-nightly
+
+on:
+  # Rebuild on merges to master
+  push:
+    branches:
+      - master
```

Comment:
Not sure if this is the best trigger. The workflow could also be a "real" nightly build to automatically include changes to druntime/phobos even when there are no changes to DMD. It also would save a lot of CI time.

Reply:
Switched to a daily build at 00:00 UTC